<?xml version="1.0" encoding="UTF-8"?>

<launch>
  
  <!-- Transform from base_footprint frame to jy901_imu frame -->
  <node pkg="tf2_ros" 
        type="static_transform_publisher" 
        name="base_footprint_to_jy901_imu" 
        args="-0.2 0 0.13 0 0 0 base_footprint jy901_imu">
  </node>

  <node pkg="imu"
        type="jy901.py" 
        name="jy901" >
    
  </node>
</launch>

